whitepapers/Tables.md

```diff
@@ -2,4 +2,40 @@
 
 
 
+## Pipeline configuration
 
+### Source
```

Comment:
Spit balling some ideas, for the admin we should throw in some aggregations.

Last harvest `SELECT * FROM harvest_log WHERE source_id = me SORT BY started LIMIT 1`
Total harvests `SELECT COUNT(*) FROM harvest_log WHERE source_id = me`
Success % ...
Etc.